app/helpers/registration_helper.rb

```diff
@@ -146,14 +146,26 @@ def generate_tracking_sheet(druid, pdf)
         pdf.y -= pdf.font.height * 1.5
       end
     end
+    pdf
   end
 
-  def extract_identifiers(xml)
-    xml.search('/mods:mods/mods:identifier | /msDesc/msIdentifier/*', { 'mods' => 'http://www.loc.gov/mods/v3' }).collect do |elem|
-      unless elem.text.empty?
-        ["#{(elem['displayLabel'] || elem.name).titleize}:", elem.text]
-      end
-    end.compact
+  # @param [Hash] doc Solr document or to_solr Hash
+  # @return [Array<Array<String>>] Complex array suitable for pdf.table()
+  def doc_to_table(doc)
+    table_data = []
+    labels = doc['obj_label_ssim']
+    label = (labels.nil? || labels.empty?) ? '' : labels.first
+    label = label[0..110] + '...' if label.length > 110
+    table_data.push(['Object Label:', label])
+    table_data.push(['Project Name:', doc['project_tag_ssim'].to_s]) if doc['project_tag_ssim']
+
+    tags = Array(doc['tag_ssim']).collect { |tag| tag =~ /^Project\s*:/ ? nil : tag.gsub(/\s+/, Prawn::Text::NBSP) }.compact
+    table_data.push(['Tags:', tags.join("\n")]) if tags.length > 0
+    table_data.push(['Catkey:',    Array(doc['catkey_id_ssim']).join(', ')]) if doc['catkey_id_ssim'].present?
+    table_data.push(['Source ID:', Array(doc['source_id_ssim']).first     ]) if doc['source_id_ssim'].present?
+    table_data.push(['Barcode:',   Array(doc['barcode_id_ssim']).first    ]) if doc['barcode_id_ssim'].present?
+    table_data.push(['Date Printed:', Time.now.strftime('%c')])
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.